README.md

```diff
@@ -34,6 +34,16 @@ Make use of the many generators for code, try `ember help generate` for more det
 * `ember test`
 * `ember test --server`
 
+### Generating Documentation
+
+You can use `ember-cli-yuidoc` to generate the documentation in the `docs/` directory:
+
+* `ember ember-cli-yuidoc`
+
+You can also automatically generate, update and serve them at[http://localhost:4200/docs/](http://localhost:4200/docs/):
```

Comment:
IMO "it" should be used instead of "them". A documentation is thing, not a group (of persons). But I think it sounds better to use "the documentation" again.